crates/cli/src/subcommands/describe.rs

```diff
@@ -20,16 +21,16 @@ pub fn cli() -> clap::Command {
                 .requires("entity_type")
                 .help("The name of the entity to describe"),
         )
-        .arg(Arg::new("brief").long("brief").short('b').action(SetTrue)
-            .help("If this flag is present, a brief description shall be returned"))
         .arg(
-            Arg::new("identity")
-                .long("identity")
-                .short('i')
-                .conflicts_with("anon_identity")
-                .help("The identity to use to describe the entity")
-                .long_help("The identity to use to describe the entity. If no identity is provided, the default one will be used."),
+            Arg::new("brief")
+                .long("brief")
+                .short('b')
+                .action(SetTrue)
+                .help("If this flag is present, a brief description shall be returned"),
```

Comment:
I can't figure out why this reformat happened now. I can't seem to retrigger it in a separate PR.